### PR TITLE
[Scenes] Fix (new?) failing scenes on the CI

### PR DESCRIPTION
**Pull request by fredroy**
I dont know why but the CI raises 3 new erroneous scenes (and rightfully so)
This PR corrects them anyway (and removes the warnings in the meantime)

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by fredroy** [comment]
[ci-build][with-all-tests]

File: examples/Tutorials/Collision/AdvancedResponseConstraintBased.scn [hunk 1]
```diff
@@ -1,56 +1,63 @@
...
     <DefaultContactManager name="Response" response="FrictionContact" />
-    <DefaultCollisionGroupManager />
     <LCPConstraintSolver maxIt="1000" tolerance="0.001" maxIterations="1000"/>
     <!-- Using a rigid cube using collision triangles, lines and points  -->
-    <Node name="Cube">
+    <Node name="Cube1">
         <EulerImplicitSolver name="EulerImplicit"  rayleighStiffness="0.1" rayleighMass="0.1" />
         <CGLinearSolver name="CG Solver" iterations="25" tolerance="1e-5" threshold="1e-5"/>
-        <MechanicalObject name="Cube RigidDOF" template="Rigid3d" dy="2" />
+        <MechanicalObject name="Cube_RigidDOF" template="Rigid3d" dy="2" />
```
Comment (epernod) [inline examples/Tutorials/Collision/AdvancedResponseConstraintBased.scn]: Cube1_RigidDOF

File: examples/Tutorials/Collision/AdvancedResponseConstraintBased.scn [hunk 2]
```diff
@@ -1,56 +1,63 @@
...
+            <RigidMapping name="RigidMapping Collision-RigidDOF" input="@../Cube_RigidDOF" output="@Collision_Cube" />
         </Node>
     </Node>
     <!-- Using a rigid cube using collision triangles, lines and points  -->
-    <Node name="Cube">
+    <Node name="Cube2">
         <EulerImplicitSolver name="EulerImplicit" />
         <CGLinearSolver name="CG Solver" iterations="25" tolerance="1e-5" threshold="1e-5"/>
-        <MechanicalObject name="Cube RigidDOF" template="Rigid3d" dy="5" dx="1" />
+        <MechanicalObject name="Cube_RigidDOF" template="Rigid3d" dy="5" dx="1" />
```
Comment (epernod) [inline examples/Tutorials/Collision/AdvancedResponseConstraintBased.scn]: vs Cube2_RigidDOF

**Review by epernod — COMMENTED** [review]
small remarks, not blocking the PR

**Pull request merged**